src/serverMain/kotlin/com/studo/campusqr/database/InitialSetup.kt

```diff
@@ -68,19 +77,86 @@ suspend fun initialDatabaseSetup() {
       insert("ldapUserDisablingIntervalMinutes", 24 * 60)
     }
 
-    // Create root user
+
+    // Create root or demo user
     with(getCollection<BackendUser>()) {
-      val rootUser = BackendUser().apply {
-        _id = "rootUser"
-        email = "admin@example.org"
-        passwordHash = Algorithm.hashPassword("admin")
-        name = "Root User"
-        createdDate = Date()
-        createdBy = _id
-        type = UserType.ADMIN
+      if (DEMO_MODE == 1) {
+        // Delete root user so there is no other way to log in
+        // TODO decide if we should just drop all users if in demo mode?
+        deleteOne(BackendUser::_id equal "rootUser")
+        val user =
+          BackendUser().apply {
```

Comment:
Why do we need a separate demoUser and can't reuse the rootUser as "demo-user"? 
If you think because of the possibility to delete all data that was created during the demo: We need for that neverless a way to automatically/manually delete all user-created data and restore some demo-data (locations, checkIns...)